### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by janitastic** [janitastic/renew-hotel#6]
- [x] Accessibility audits should be at 100% for the dashboard.
- [x] A user should be able to interact with all functionality of your application by tabbing through it, no use of the trackpad
- [ ] ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
